src/elliptic/curves/secp256_k1.rs

```diff
@@ -543,9 +543,8 @@ impl Serialize for Secp256k1Point {
     where
         S: Serializer,
     {
-        let mut state = serializer.serialize_struct("Secp256k1Point", 2)?;
-        state.serialize_field("x", &self.x_coor().unwrap().to_hex())?;
-        state.serialize_field("y", &self.y_coor().unwrap().to_hex())?;
+        let mut state = serializer.serialize_struct("Secp256k1Point", 1)?;
+        state.serialize_field("p", &self.bytes_compressed_to_big_int().to_hex())?;
```

Comment:
1) Since serialized structure `Secp256k1Point` consist of only one field, maybe just serialize it as string, not as structure? Ie. simple [`serializer.serialize_str(...)`] can be used here
2) In curv v0.8 we [serialize][curv08-ser] point as raw bytes, not as hex-encoded bytes. This removes verbosity, but makes it highly efficient for binary data formats like `bincode`. Simply saying, it reduces size of serialized point by half. Do you prefer hex format?

[`serializer.serialize_str(...)`]: https://docs.rs/serde/1.0.130/serde/ser/trait.Serializer.html#tymethod.serialize_str
[curv08-ser]: https://github.com/ZenGo-X/curv/blob/9892ca7969778a5d3d2c3a596c9e9298ccc9cbb9/src/elliptic/curves/wrappers/serde_support.rs#L22-L26

Reply:
1. True, it is shorter and simpler than my idea about newtype struct.
2. As a matter of fact, I prefer raw bytes as well. I'll change the code soon.